product_docs/docs/edbcloud/beta/getting_started/01_preparing_azure/01_understanding_qotas_in_azure.mdx

```diff
@@ -0,0 +1,68 @@
+---
+title: "Understanding resource requirements"
+
+---
+
+By default, Azure sets low resource limits on the number of vCPUs (cores), and Public IP addresses permitted in your subscription. The following sections explain BigAnimal's requirements and recommended resource limits in Azure.
+ 
+### Public IP addresses limits
+
+The number of Public IP addresses must not exceed the limit for a particular region.
+
+##### Default Azure limit
+
+The default Public IP addresses limits for **Basic** and **Standard** type are set to 10. See [Public IP address limits](https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/azure-subscription-service-limits#publicip-address) for more information.
+
+##### Recommended limit
+
+EDB recommends you to increase the Public IP addresses limits for **Basic** and **Standard** type to the number of clusters you plan to deploy in a given region.
+
+### vCPU limits
+
+Any time a new VM is deployed in Azure, the vCPUs for the VMs must not exceed the vCPU limits for the VM size family or the total regional vCPU limit. BigAnimal uses Standard Dsv2 Family vCPUs (cores) to manage workloads and Standard Esv3 Family vCPUs (cores) to manage clusters.
```

Comment:
https://enterprisedb.atlassian.net/browse/UPM-1991 and https://enterprisedb.atlassian.net/browse/UPM-2667 are both scheduled to land in November and will require a complete rewrite of this section.